CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+## Release 1.7.1
+
+This is a minor patch release.
+
+Bug fixes and minor improvements:
+- client: Fix uptime sort in table view
+	[#3038](https://github.com/weaveworks/scope/pull/3038)
+- Check if k8s resources are supported in `runReflectorUntil`
+	[#3037](https://github.com/weaveworks/scope/pull/3037)
+- logReadCloser: ensure EOF after `Close()`
+	[#3034](https://github.com/weaveworks/scope/pull/3034)
+- Stop page router on App unmount
+	[#3025](https://github.com/weaveworks/scope/pull/3025)
```

Comment:
The two major issues should come first and also deserve explanation in the highlights, i.e. saying what problems they will have caused.